front/app/containers/Admin/projects/project/admin_phase_email_wrapper/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+
+import useCampaigns from 'api/campaigns/useCampaigns';
+import useSupportedCampaignNames from 'api/campaigns/useSupportedCampaignNames';
+
+import useLocalize from 'hooks/useLocalize';
+
+import CampaignRow from 'containers/Admin/messaging/AutomatedEmails/CampaignRow';
+import { stringifyCampaignFields } from 'containers/Admin/messaging/AutomatedEmails/utils';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+
+const AdminPhaseEmailWrapper = () => {
+  const localize = useLocalize();
+  const { phaseId } = useParams();
+  const supportedCampaignNames =
+    useSupportedCampaignNames({ phaseId }).data?.data.attributes || [];
+  const contextCampaigns = useCampaigns({
+    ...(phaseId ? { phaseId } : {}),
+    pageSize: 250,
+  }).data?.pages.flatMap((page) => page.data);
+  const globalCampaigns = useCampaigns({ pageSize: 250 }).data?.pages.flatMap(
+    (page) => page.data
+  );
+  return (
+    <Box>
+      <Text color="coolGrey600" mt="0px" fontSize="m">
+        <FormattedMessage {...messages.automatedEmailsDescription} />
+      </Text>
+      {supportedCampaignNames.length > 0 &&
+        supportedCampaignNames.map((campaignType) => {
+          let campaign = contextCampaigns?.find(
+            (campaign) => campaign.attributes.campaign_name === campaignType
+          );
+          if (!campaign) {
+            campaign = globalCampaigns?.find(
+              (campaign) => campaign.attributes.campaign_name === campaignType
+            );
+          }
+
+          return (
+            campaign && (
+              <CampaignRow
+                campaign={stringifyCampaignFields(campaign, localize)}
+                key={campaign.id}
+                phaseId={phaseId}
+                // onClickViewExample={onClickViewExample(campaign.id)}
```

Comment:
Leftover comment? :)

Reply:
I was considering to enable it somewhere in the future of this tandem, but we might not even have the time for it. I'll just remove it.